cmrtlib/linux/hardware/drm_device.h

```diff
@@ -39,16 +39,16 @@
 #include <limits.h>
 #include <signal.h>
 #include <time.h>
-#include <sys/sysmacros.h>   //<sys/types.h>
+#include <sys/types.h>
 #include <sys/stat.h>
 #define stat_t struct stat
 #include <sys/ioctl.h>
 #include <sys/time.h>
 #include <stdarg.h>
-#ifdef MAJOR_IN_MKDEV
+#ifdef __sun //#ifdef MAJOR_IN_MKDEV
 #include <sys/mkdev.h>
 #endif
-#ifdef MAJOR_IN_SYSMACROS
+#if defined(__GLIBC__) || defined(__linux__) //#ifdef MAJOR_IN_SYSMACROS
```

Comment:
Why is this change correct?

Reply:
Because the bundled and patched libdrm is built via CMake and doesn't define `MAJOR_IN_SYSMACROS`. Instead always-true conditionals on Linux were expanded while the rest left as is. Are you suggesting to implement the macros via CMake?

Reply:
Let's see if CMake checks pass CI.

Reply:
*Sigh* CI is blocked by `1 workflow awaiting approval`. This hasn't been the case when TravisCI was still operational a year ago.

Reply:
Oh, I see. I didn't realize this was a bundled libdrm file. Ugh.

Never mind then.

Reply:
OK, reverted CMake checks. GNU libc disables extensions by default, so it breaks configure checks without `_GNU_SOURCE`. Debugging without a Linux machine/VM maybe hard.
```c++
-- Looking for major
-- Looking for major - not found
-- Looking for major
-- Looking for major - not found
[...]
In file included from cmrtlib/linux/hardware/cm_device_os.cpp:24:
cmrtlib/linux/../linux/hardware/drm_device.h:1041:11: error: use of undeclared identifier 'major'
    maj = major(sbuf.st_rdev);
          ^
cmrtlib/linux/../linux/hardware/drm_device.h:1042:11: error: use of undeclared identifier 'minor'
    min = minor(sbuf.st_rdev);
          ^
```
https://github.com/jbeich/media-driver/runs/4452912703